tests/bin/cloud_health_to_graphite_test.py

```diff
@@ -40,25 +41,40 @@ class CloudHealthAPITest(unittest.TestCase):
     REPORT_NAME = re.sub('[ \.]+', '_', REPORT_NAME_ARG)
     SET_DATE = '2016-05-01'
     DATE_FORMAT = '%Y-%m-%d %H:%M'
+    INTERVAL = Interval.daily
 
     _DEFAULT_DATE_FORMAT = '%Y-%m-%d'
+    _DEFAULT_TIME_INTERVAL = Interval.hourly
     _STDOUT_FORMAT = 'cloud_health.{env}.{report_name}.{category} {cost} {date}\n'
 
     @staticmethod
-    def _get_cloud_health_return(report_id, category=None, date_format=_DEFAULT_DATE_FORMAT):
+    def _get_cloud_health_return(
+        report_id,
+        category=None,
+        date_format=_DEFAULT_DATE_FORMAT,
+        time_interval=_DEFAULT_TIME_INTERVAL,
+    ):
         """
         Creates a fake data to mock Cloud Health API
         """
         result = {
             'Total': {'key1': 'value1', 'key2': None},
         }
+
         dt = datetime.today()
+        dt_diff = {
+            Interval.monthly: {'days': 30},
```

Comment:
You are correct. But since this is a unit test, I am willing to take that risk for maintainability and readability. The biggest thing here is to test the datetime format. Adding a comment for this.